docs/multi-agent-manager.md

```diff
@@ -52,7 +60,7 @@ manager.add_agent(weather_station_id, agent_overrides=agent_overrides)
 component_overrides = {
     "name": "p2p_libp2p",
     "author": "fetchai",
-    "version": "0.9.0",
+    "version": "0.13.0",
     "type": "connection",
```

Comment:
@DavidMinarsch this wasn't detected by the bump script... we should update the script to detect such JSON objects.

Reply:
I think the bump script only locks at current vs next version. so it probably will be picked up going forward..